src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -857,12 +862,21 @@ namespace winrt::TerminalApp::implementation
                                  WI_IsFlagSet(lShiftState, CoreVirtualKeyStates::Down) ||
                                  WI_IsFlagSet(rShiftState, CoreVirtualKeyStates::Down) };
 
+        const auto ctrlState{ window.GetKeyState(VirtualKey::Control) };
+        const auto rCtrlState = window.GetKeyState(VirtualKey::RightControl);
+        const auto lCtrlState = window.GetKeyState(VirtualKey::LeftControl);
+        const auto ctrlPressed{ WI_IsFlagSet(ctrlState, CoreVirtualKeyStates::Down) ||
+                                WI_IsFlagSet(rCtrlState, CoreVirtualKeyStates::Down) ||
+                                WI_IsFlagSet(lCtrlState, CoreVirtualKeyStates::Down) };
```

Comment:
I always found this code curious... Reading the docs again:
> The Ctrl key or button. This is the general Ctrl case, applicable to key layouts with only one Ctrl key or that do not need to differentiate between left Ctrl and right Ctrl keystrokes.

It seems to me as if this is enough:
```cpp
const auto ctrlPressed = WI_IsFlagSet(window.GetKeyState(VirtualKey::Control), CoreVirtualKeyStates::Down);
```

(Same for Shift and Menu.)

Reply:
Honestly, I don't know. I've been copy pasting that same structure of code for years since the first Terminal prototypes. No idea if it's really all necessary.

Reply:
let's file a followup to clean these up, k?